src/safeds/data/tabular/containers/_table.py

```diff
@@ -695,6 +697,34 @@ def get_row(self, index: int) -> Row:
 
         return Row._from_pandas_dataframe(self._data.iloc[[index]], self._schema)
 
+    def _get_similar_columns(self, column_name: str) -> list[str]:
+        """
+        Get all similar column names to the given one that's not in the table.
+
+        Parameters
+        ----------
+        column_name : str
+            The name of the Column, that's not in the Table.
+
+        Returns
+        -------
+        similar_columns: list[str]
+            A list of all similar column names.
```

Comment:
Maybe be a bit more exact here:
```suggestion
            A list of all column names in the Table that are similar or equal to the given column name.
```